scenariorun/src/main/openapi/scenariorun.yaml

```diff
@@ -6,17 +6,37 @@ info:
   version: 0.0.8-SNAPSHOT
 
 servers:
-- url: 'https://api.azure.cosmo-platform.com'
-- url: 'http://localhost:4010'
+  - url: 'https://api.azure.cosmo-platform.com'
+  - url: 'http://localhost:4010'
 
 security:
-- oAuth2AuthCode: [ ]
+  - oAuth2AuthCode: [ ]
 
 tags:
-- name: scenariorun
-  description: ScenarioRun Management
+  - name: scenariorun
+    description: ScenarioRun Management
 
 paths:
+  /organizations/{organization_id}/scenarioruns/deletehistoricaldata:
+    parameters:
+      - name: organization_id
+        in: path
+        description: the Organization identifier
+        required: true
+        schema:
+          type: string
+    delete:
```

Comment:
Either:
* the endpoint is deletehistoricaldata and the verb is POST
* the endpoint is historicaldata and the verb is DELETE
OK for this version.

Reply:
done